### ISSUE TITLE
Lumen [5.3] Not working

### ISSUE THREAD

**Issue by computerfr33k** [flugger/laravel-responder#22]
```php
BindingResolutionException in Container.php line 763:
Target [Illuminate\Contracts\Routing\ResponseFactory] is not instantiable.
in Container.php line 763
at Container->build('Illuminate\Contracts\Routing\ResponseFactory', array()) in Container.php line 644
at Container->make('Illuminate\Contracts\Routing\ResponseFactory', array()) in Application.php line 211
at Application->make('Illuminate\Contracts\Routing\ResponseFactory') in Container.php line 1203
at Container->offsetGet('Illuminate\Contracts\Routing\ResponseFactory') in ResponderServiceProvider.php line 121
at ResponderServiceProvider->Flugg\Responder\{closure}(object(Application), array()) in Container.php line 746
at Container->build(object(Closure), array()) in Container.php line 644
at Container->make('Flugg\Responder\Http\SuccessResponseBuilder', array()) in Application.php line 211
at Application->make('Flugg\Responder\Http\SuccessResponseBuilder') in Container.php line 1203
at Container->offsetGet('Flugg\Responder\Http\SuccessResponseBuilder') in ResponderServiceProvider.php line 74
at ResponderServiceProvider->Flugg\Responder\{closure}(object(Application), array()) in Container.php line 746
at Container->build(object(Closure), array()) in Container.php line 644
at Container->make('Flugg\Responder\Responder', array()) in Application.php line 211
at Application->make('Flugg\Responder\Responder', array()) in helpers.php line 39
at app('Flugg\Responder\Responder') in helpers.php line 14
at responder() in web.php line 23
at Closure->{closure}()
at call_user_func_array(object(Closure), array()) in Container.php line 508
at Container->call(object(Closure), array()) in RoutesRequests.php line 523
at Application->callActionOnArrayBasedRoute(array(true, array(object(Closure)), array())) in RoutesRequests.php line 497
at Application->handleFoundRoute(array(true, array(object(Closure)), array())) in RoutesRequests.php line 394
at Application->Laravel\Lumen\Concerns\{closure}() in RoutesRequests.php line 650
at Application->sendThroughPipeline(array(), object(Closure)) in RoutesRequests.php line 400
at Application->dispatch(null) in RoutesRequests.php line 341
at Application->run() in index.php line 28
```

**Comment by computerfr33k** [comment]
Seems Lumen 5.1 is also not working.

**Comment by flugger** [comment]
Apparently, Lumen uses its own version of the response factory. Fixed in `1.2.11`, thanks! https://github.com/flugger/laravel-responder/commit/3009e55cd813f8db5e0c0aea1e7c7a5648798837
